tests/local-peers.js

```diff
@@ -217,6 +217,7 @@ test('Invite to unknown peer', async (t) => {
       roleName: DEFAULT_CAPABILITIES[MEMBER_ROLE_ID].name,
       invitorName: 'device0',
     }),
+    // @ts-ignore
```

Comment:
I think we can avoid this `ts-ignore` by updating `types/brittle.d.ts`. We can replace our old `ExceptionAssertion` with this:

```typescript
type AnyErrorConstructor = new () => Error

interface ExceptionAssertion {
  (fn: () => unknown, message?: string): void
  (fn: () => unknown, error?: RegExp | AnyErrorConstructor, message?: string): void
  (fn: Promise<unknown>, message?: string): Promise<void>
  (fn: Promise<unknown>, error?: RegExp | AnyErrorConstructor, message?: string): Promise<void>
  all(fn: () => unknown, message?: string): void
  all(fn: () => unknown, error?: RegExp | AnyErrorConstructor, message?: string): void
  all(fn: Promise<unknown>, message?: string): Promise<void>
  all(fn: Promise<unknown>, error?: RegExp | AnyErrorConstructor, message?: string): Promise<void>
}
```

That should let us remove these.

Reply:
Great, thanks for fixing this!